TEI.js

```diff
@@ -109,6 +109,27 @@ var exportedXMLIds = {};
 var generatedItems = {};
 var allItems = {};
 
+// replace formatting with TEI tags
+function replaceFormatting (title){
+            var titleText = title;
+            // italics
+            titleText = titleText.replace('<i>', '<hi rend="italics">');
+            titleText = titleText.replace('</i>', '</hi>');
+            // bold
+            titleText = titleText.replace('<b>', '<hi rend="bold">');
+            titleText = titleText.replace('</b>', '</hi>');
+            // subscript
+            titleText = titleText.replace('<sub>', '<hi rend="sub">');
+            titleText = titleText.replace('</sub>', '</hi>');
+            // superscript
+            titleText = titleText.replace('<sup>', '<hi rend="sup">');
+            titleText = titleText.replace('</sup>', '</hi>');
+            // small caps
+            titleText = titleText.replace('<span style="font-variant:small-caps;">', '<hi rend="smallcaps">');
```

Comment:
this may be a little too strict in terms of possible whitespace. e.g. "font-variant: small-caps;" but I think we would accept this.